assets/stylesheets/mobile/chat-message.scss

```diff
@@ -12,6 +12,7 @@
   @include unselectable;
 }
 
+.chat-messages-scroll .chat-messages-container,
```

Comment:
we sadly have two "chat-messages-container" element in code atm so I'm forced to declare the hierarchy